src/infrastructure/admin.py

```diff
@@ -36,32 +45,171 @@ class TechnologyAdmin(admin.ModelAdmin):
     ordering = ("title",)
 
 
+class ManyToManyCommaWidget(widgets.ManyToManyWidget):
+    def __init__(self, model, field="title", separator=","):
+        super().__init__(model, field=field, separator=separator)
+
+    def clean(self, value, row=None, *args, **kwargs):
+        if isinstance(value, str):
+            return [item.strip() for item in value.split(self.separator)]
+        return value
+
+
+class ProjectResource(resources.ModelResource):
+    user_id = fields.Field(
+        column_name="user_id",
+        attribute="user",
+        widget=widgets.ForeignKeyWidget(User, "id"),
+        default=None,
+    )
+
+    industries = fields.Field(
+        attribute="industries",
+        widget=ManyToManyCommaWidget(Industry, field="title"),
+    )
+    technologies = fields.Field(
+        attribute="technologies",
+        widget=ManyToManyCommaWidget(Technology, field="title"),
+    )
+
+    class Meta:
+        model = Project
+        fields = (
+            "user_id",
+            "title",
+            "description",
+            "industries",
+            "technologies",
+        )
+
+        export_order = (
+            "id",
+            "title",
+            "description",
+            "created_at",
+            "updated_at",
+            "user__name",
+            "user__email",
+            "user__company__title",
+            "user__date_joined",
+            "industries",
+            "technologies",
+        )
+
+    def get_import_id_fields(self):
+        return ["user_id", "title"]
+
+    def before_import(self, dataset, **kwargs):
+        super().before_import(dataset, **kwargs)
+        dataset.append_col([kwargs.get("user_id")], header="user_id")
+
+    def before_import_row(self, row, row_number=None, **kwargs):
```

Comment:
I think it would be better if you will add little comments about what is happening in this function